src/js/components/summer_club/index.js

```diff
@@ -36,12 +34,60 @@ class SummerClub extends React.Component {
     this.setState({[event.target.name]: event.target.value});
   }
 
+  validateFormData = ({parent, children}) => {
+    const errors = {};
+
+    const validator = new Validator(parent);
+    const {
+      email,
+      center,
+      phone_number,
+      name
+    } = parent
+
+    if (!validator.validateEmail(email)) {
+      errors.email = "Please enter a valid email address"
+    }
+
+    if (center === "") {
+      errors.center = "Please choose a center close to you"
+    }
+
+    if (name === "" || name.indexOf(' ') == -1) {
+      errors.name = "Please enter your full name"
+    }
+
+    if (isNaN(phone_number) || phone_number.length !== 11) {
```

Comment:
This could be an issue if the user adds the international format of a phone number. Let us just validate for the presence here to avoid complications.

Reply:
got it